docs/serving/samples/cloudevents/cloudevents-go/README.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Cloud Events - Go"
+linkTitle: "Go"
+weight: 1
+type: "docs"
+---
+
+A simple web app written in Go that can receive and send Cloud Events that you
+can use for testing.  It supports running in two modes:
+1. The default mode has the app reply to your input events with the output
+  event, which is simplest for demonstrating things working in isolation,
+  but is also the model for working for the Knative Eventing `Broker` concept.
+
+2. `SINK` mode has the app send events to the destination encoded in `$SINK`,
```

Comment:
the source contract us using `K_SINK`. We need to change it here or change the spec for sources.

Reply:
Done